src/marvin/config.py

```diff
@@ -80,6 +80,7 @@ def export_to_env_file(self):
     embeddings_cache_warn_size: int = 4000000000  # 4GB
 
     # OPENAI
+    openai_max_tokens_per_message: int = 4096
```

Comment:
This would be better as a lookup depending on the model name, not as a setting. Otherwise it becomes possible to set the model and max tokens to incompatible values. For example, a `utilities.llms.get_context_size()` utility function that returns the context given the model name (or a fallback default)